src/main/solution_space/measured_vertex/measured_vertex_v_1.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
+from __future__ import annotations
+
+from typing import Optional
+
+from src.main.solution_space.data_classes import Profile, User
+from src.main.solution_space.measured_vertex.measured_vertex import IMeasuredVertex
+from src.main.solution_space.path_finder.path_finder import log
+from src.main.solution_space.vertex import Vertex
+from src.main.util.log_util import log_and_raise_error
+
+
+class MeasuredVertexV1(IMeasuredVertex):
+
+    # Todo: get actual vertex profile
+    @classmethod
```

Comment:
Is it a **classmethod**, but with the **self** pram?

Reply:
My fault, sorry